Origin: Assassin <assassin@assasinate-you.net>
Date: Mon, 8 Feb 2021
Author: Scarlett Moore <sgmoore@kde.org>
Subject: Fix SO Version
Description: Fix the version to generate correct SO Version so that
symbols can be generated and shlibs can correctly do it's job.
Applied-Upstream: https://github.com/azubieta/xdg-utils-cxx/pull/17


diff --git a/CMakeLists.txt b/CMakeLists.txt
index 6d22314..8dab65a 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,6 +1,6 @@
 cmake_minimum_required(VERSION 3.0)
 project(XdgUtils)
-set(PROJECT_VERSION 0.1.1)
+set(PROJECT_VERSION 1.0.1)
 set(PROJECT_DESCRIPTION "Freedesktop standards C++ implementation")
 set(PROJECT_HOMEPAGE_URL https://github.com/azubieta/xdg-utils-cxx)
 
diff --git a/cmake/module_utils.cmake b/cmake/module_utils.cmake
index 2e7fe19..a615aa9 100644
--- a/cmake/module_utils.cmake
+++ b/cmake/module_utils.cmake
@@ -23,6 +23,7 @@ function(add_xdgutils_module name srcs)
         PREFIX "lib${PROJECT_NAME}"
         PUBLIC INTERFACE_LINK_DIRECTORIES $<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}>
         PUBLIC INTERFACE_LINK_DIRECTORIES $<INSTALL_INTERFACE:${PROJECT_INSTALL_LIBDIR}>
+        VERSION ${PROJECT_VERSION}
     )
 endfunction()
 
